### DISCUSSION TITLE
New contributions!

### DISCUSSION THREAD

**Discussion by bglamadrid** [trebol-ecommerce/trebol-backend-monolith#155] [Announcements]
I'll be indicating new contributors here. 

If you help this project in any way (not only code) __you can also include yourself__! See [this guide](https://allcontributors.org/docs/en/bot/overview) for how to do it, it's easy!

**Comment by bglamadrid** [comment]
@all-contributors please add @NyorJa for code

[a comment by an automated account is not shown]

**Comment by bglamadrid** [comment]
@all-contributors please add @mepox for code

[a comment by an automated account is not shown]

**Comment by mepox** [comment]
@bglamadrid Thank you!

**Comment by bglamadrid** [comment]
@all-contributors please add @NyorJa for test

[a comment by an automated account is not shown]

**Comment by bglamadrid** [comment]
@all-contributors please add @mepox for maintenance and ideas

[a comment by an automated account is not shown]

**Comment by mepox** [comment]
> @all-contributors please add @mepox for maintenance and ideas

Thank you @bglamadrid !

**Comment by bglamadrid** [comment]
@all-contributors please add @NyorJa for ideas. I almost started writing several hundred lines of boilerplate, yet Nyorja enlightened me with an existing feature of Project Lombok that I didn't realize existed.

[a comment by an automated account is not shown]
